### PR TITLE
fix: avatar has wrong aspect-ratio

### PR DESCRIPTION
**Pull request by krishguptadev**
New Pic, I like this one the most, can suggest something else in the comments

240579221_614648419505322_8319929986706745115_n

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/828"></a>

### CODE REVIEW

**Review by Panquesito7 — COMMENTED** [review]
Not sure about this, maybe another cool profile photo could be set 🤷‍♂️ Let's wait for Eddie's review. :)

File: public/data/eddieandsaraexplore.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "Eddie and Sara Explore",
   "type": "personal",
   "bio": "Digital Nomads living in the present for the stories we will tell in the future",
-  "avatar": "https://user-images.githubusercontent.com/624760/146668623-7d4363ac-0713-49e2-a087-ce3249046068.jpg",
+  "avatar": "https://user-images.githubusercontent.com/91655303/146677058-374e7086-da49-4c85-bf38-db3d1030ccee.jpg",
```
Comment (krishguptadev) [inline public/data/eddieandsaraexplore.json:5]:
```suggestion
  "avatar": "https://user-images.githubusercontent.com/91655303/146702057-60099877-49b6-4cd4-aa0c-0b050e8b1216.jpg",
```

Comment (krishguptadev) [reply public/data/eddieandsaraexplore.json]: I found a better one

**Comment by eddiejaoude** [comment]
Good spot, the photo needs to be square 👍

**Comment by eddiejaoude** [comment]
I think this photo will have the same issue though

**Comment by eddiejaoude** [comment]
```
https://user-images.githubusercontent.com/624760/146711385-4aabecfb-fe1e-4d12-8b20-008b4c6a085b.jpg
```

DSC02421-2

**Comment by krishguptadev** [comment]
> I think this photo will have the same issue though

I think I cropped it, might have uploaded the wrong one sorry!

File: public/data/eddieandsaraexplore.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "Eddie and Sara Explore",
   "type": "personal",
   "bio": "Digital Nomads living in the present for the stories we will tell in the future",
-  "avatar": "https://user-images.githubusercontent.com/624760/146668623-7d4363ac-0713-49e2-a087-ce3249046068.jpg",
+  "avatar": "https://user-images.githubusercontent.com/91655303/146702057-60099877-49b6-4cd4-aa0c-0b050e8b1216.jpg",
```
Comment (krishguptadev) [inline public/data/eddieandsaraexplore.json:5]:
```suggestion
  "avatar": "https://user-images.githubusercontent.com/624760/146711385-4aabecfb-fe1e-4d12-8b20-008b4c6a085b.jpg",
```

**Review by Panquesito7 — APPROVED** [review]

**Review by Vyvy-vi — APPROVED** [review]
lgtm! Though, I wonder if moving forward we could use a different place for fetching images from, rather than github's servers

**Pull request merged**